### ISSUE TITLE
Date is returned as number

### ISSUE THREAD

**Issue by leosurban** [nukedzn/node-informix#38]
Date is returned as number, probably as number of days starting at 1900.
Will be better to get JS DATE and null.

dbaccess:
>echo select dtmprec from cenmat where nazevmater like 'AR%' | dbaccess pemaxdb
Database selected.
dtmprec
02.12.2016
16.05.2016
18.08.2016
17.05.2016
// nothing i.m. NULL
// nothing i.m. NULL
06.12.2016
7 row(s) retrieved.

node-informix:
[ [ 42705 ],
  [ 42505 ],
  [ 42599 ],
  [ 42506 ],
  [ -2147483648 ],
  [ -2147483648 ],
  [ 42709 ] ]

```
var opts = {
  database : 'pemaxdb@ixlur',
  username : 'informix',
  password : '***'
};
var informix = require( 'informix' )( opts );
informix
  .query( "select dtmprec from cenmat where nazevmater like 'AR%';" )
  .then( function ( cursor ) 
  {
    return  cursor.fetchAll();    
  } )
  .then( function ( results )
  {
     console.log( results );
  })
  .catch( function ( err ) 
  {
    console.log( err );
  } );
```

**Comment by uditha-atukorala** [comment]
@leosurban, can you provide sample schema and insert statements to replicate the issue please? 

Datetime fields should be returned as ISO formatted date strings (e.g. "2017-02-17T17:20:56.002Z") which seems to work for me (see [commit #5b65fb8](https://github.com/nukedzn/node-informix/commit/5b65fb894316a455483e60512f8c3fd959b11871)).

**Comment by leosurban** [comment]
Here is table with DATE field. 

```
create table exampletable
(
  id integer not null,
  somename nvarchar(100) not null,
  somedate date not null,
  primary key(id)
);

```

For February 18, 2017 is returned 42783.
It is number of days from December 31, 1899.
https://www.timeanddate.com/date/duration.html
![todaydate](https://cloud.githubusercontent.com/assets/25840153/23091785/62fbbcc0-f5be-11e6-9161-c9c2088194ef.jpg)

Thanks.

**Comment by leosurban** [comment]
Workaround for now:
```
var opts = {
  database : 'pemaxdb@ixlur',
  username : 'informix',
  password : '***'
};
var informix = require( 'informix' )( opts );
var moment = require('moment');
informix
  .query( "select dtmprec from cenmat where nazevmater like 'AR%'" )
  .then( function ( cursor ) 
  {
    return  cursor.fetchAll();
    
  } )
  .then( function ( results )
  {
   for (var i=0; i<results.length; i++)
   {
     var dtmprec = null;
     if ( results[i][0] != -2147483648 ) // is not null
     { 
        // npm install moment
        // var moment = require('moment');
        var baseDate = moment("1899-12-31");
        dtmprec = baseDate.add( results[i][0], 'days' ).calendar(); 
     }     
     console.log( dtmprec );
   }
  })
  .catch( function ( err ) 
  {
    console.log( err );
  } );
```
Result:
```
12/02/2016
05/16/2016
08/18/2016
05/17/2016
null
null
12/06/2016
```

**Comment by uditha-atukorala** [comment]
Published v0.5.6 which returns `DATE` fields as ISO date strings (similar to `DATETIME` fields) or empty string ('') if `null`.
